medor_custom/controllers/user.py

```diff
@@ -36,14 +33,12 @@ def user_form(self, **kw):
             if user.parent_id:
                 # company
                 company = user.parent_id
-                company.write({
-                    'name': request.params['company_name'],
-                })
+                company.write({"name": request.params["company_name"]})
                 try:
-                    user.vat = request.params['vat']
+                    user.vat = request.params["vat"]
                 except ValidationError as err:
-                    request.params['error'] = err.name
-                if request.params.get('invoice_address', False):
+                    request.params["error"] = err.name
+                if request.params.get("invoice_address", False):
```

Comment:
Try to do formatting improvements in separate commits, it makes the review more readable.

Reply:
You're right, sorry. Thanks for reviewing. :)